ansible/group_vars/all

```diff
@@ -166,10 +166,12 @@ nginx:
     adminportal: 8443
   ssl:
     path: "{{ openwhisk_home }}/ansible/roles/nginx/files"
-    cert: "openwhisk-cert.pem"
-    key: "openwhisk-key.pem"
+    cert: "openwhisk-server-cert.pem"
+    key: "openwhisk-server-key.pem"
     password_enabled: false
     password_file: "ssl.pass"
+    client_ca_cert: "openwhisk-client-ca-cert.pem"
+    verify_client: "optional"
```

Comment:
@ningyougang You can write it out to `whisk.properties` and read it from there using the `WhiskProperties` singleton.